api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -611,6 +611,40 @@ type FelixConfigurationSpec struct {
 	// [Default: Auto]
 	BPFConntrackCleanupMode *BPFConntrackMode `json:"bpfConntrackMode,omitempty" validate:"omitempty,oneof=Auto Userspace BPFProgram"`
 
+	// BPFConntrackTimers overides the default values for the specified conntrack timer if
+	// set. It is a key-value make, where each value can be either a duration or `auto` to
+	// pick the value from a Linux conntrack timeout.
+	//
+	// Possible values for the keys are: CreationGracePeriod, TCPPreEstablished,
+	// TCPEstablished, TCPFinsSeen, TCPResetSeen, UDPLastSeen, GenericIPLastSeen,
+	// ICMPLastSeen.
+	//
+	// Unset or incorrect values are replaced by the default values with a warning log for
+	// incorrect values.
+	//
+	// Current auto mappings:
+	//
+	// TCPPreEstablished: nf_conntrack_tcp_timeout_syn_sent
+	// TCPEstablished:    nf_conntrack_tcp_timeout_established
+	// TCPFinsSeen:       nf_conntrack_tcp_timeout_time_wait
+	// GenericIPLastSeen: nf_conntrack_generic_timeout
+	// ICMPLastSeen:      nf_conntrack_icmp_timeout
+	//
+	// If there is no mapping, 'auto' is replaced by the default value.
```

Comment:
Now that we have defined these as pointers I don't think we need a special "auto" field - `nil` means "use the default value", right? Or am I misunderstanding the semantics of "auto"?

Reply:
Yes we need auto to distinguish between "use the default" aka nil/unset and use the value from linux if available aka auto

Reply:
Got it - in that case, I think the description here isn't clear enough. That behavior wasn't obvious to me. I'll add a suggested edit.